test/functional/p2p_feefilter.py

```diff
@@ -48,6 +37,10 @@ def on_inv(self, message):
             if (i.type == MSG_TX):
                 self.txinvs.append(hashToHex(i.hash))
 
+    def wait_for_invs_to_match(self, test_framework, invs_expected):
+        test_framework.wait_until(lambda: sorted(invs_expected) == sorted(self.txinvs),
```

Comment:
I wonder if we can eek some more perf by memoizing (untested):
```suggestion
        expected_txinvs = sorted(invs_expected)
        test_framework.wait_until(lambda: sorted(self.txinvs) == expected_txinvs),
```

Reply:
Yes, that makes to do the sorting of the txinvs to test for only once. Did it with in-place sort of the passed list.